editor/test/integration/collection_test.clj

```diff
@@ -260,8 +260,8 @@
 (deftest component-ddf-scale
   (letfn [(make-components! [game-object sprite-resource atlas-resource app-view]
             (let [sprite-resource-type (resource/resource-type sprite-resource)
-                  embedded-component (test-util/add-embedded-component! app-view sprite-resource-type game-object)
-                  referenced-component (test-util/add-referenced-component! app-view sprite-resource game-object)]
+                  embedded-component (test-util/add-embedded-component! game-object sprite-resource-type)
+                  referenced-component (test-util/add-referenced-component! game-object sprite-resource)]
```

Comment:
The `test-util/add-embedded-component!` and `test-util/add-referenced-component!` functions no longer take an `app-view` argument, and make no attempts to update the selection in the `app-view`. Also, the `game-object` was moved to the first argument position.